### DISCUSSION TITLE
Troubleshooting 2D im2im labelfree image prediction

### DISCUSSION THREAD

**Discussion by KartikeyKansal1** [AllenCellModeling/cyto-dl#316] [General]
Hi Allen Institute for Cell Science,
I'm Kartikey Kansal (kkansal2@wisc.edu), a PhD student at Department of Biomedical Engineering, University of Wisconsin-Madison. I’m working on a project to answer questions about cell mechanics and I'm trying to use Cyto-DL as one of my tools. 

I have been posting my troubleshooting comments on this thread: https://github.com/AllenCellModeling/cyto-dl/discussions/314, though the process so far has been a bit time consuming. I've followed the documentation so far and all the previous errors are resolved but I still don't see any predicted images in logs or save directory. I was wondering if someone at the institute would be interested to setup a zoom call with me and my advisor Dr. Joshua M Brockman to discuss troubleshooting and potential collaboration.  Appreciate the help so far in resolving errors!

**Comment by benjijamorris** [comment]
Hi Kartikey, thanks again for your patience as we develop this new tool! I'll be reaching out via email today.
